dask/blockwise.py

```diff
@@ -190,6 +183,11 @@ class Blockwise(Layer):
         to materialize the low-level graph).
     annotations: dict (optional)
         Layer annotations
+    io_deps: set (optional)
+        Set of "placeholder" collection names that will be generated within
+        this Blockwise layer. Since these collections do not actually exist
+        outside the layer, any key with a name in this set will be excluded
+        from the external dependencies.
```

Comment:
Do we still need `io_deps`?

Reply:
It may be possible to remove the need for this, but it would require a more-significant change to the existing logic in Blockwise.  It hink the IO-function wrappers would need to accept an integer (or tuple of integers), and would need to handle the fact that these args are not actually keys.  It seemed simpler (and more intuitive) to pass actual keys to the IO-function wrappers, and simply use the `io_deps` set to make sure these keys are never treated as "real" dependencies.